app/controllers/test/oidc_test_controller.rb

```diff
@@ -0,0 +1,154 @@
+require './spec/support/oidc_auth_helper'
+module Test
+  class OidcTestController < ApplicationController
+    include OidcAuthHelper
+
+    BIOMETRIC_REQUIRED = 'biometric-comparison-required'
+
+    def initialize
+      @client_id = 'urn:gov:gsa:openidconnect:sp:test'
+      @update_redirect_uri = true
+      super
+    end
+
+    def index
+      # default to require
+      @start_url_selfie = "#{test_oidc_auth_request_url}?ial=biometric-comparison-required"
+      @start_url_ial2 = "#{test_oidc_auth_request_url}?ial=2"
+      @start_url_ial1 = "#{test_oidc_auth_request_url}?ial=1"
+      update_service_provider if @update_redirect_uri
```

Comment:
this is always true? should we remove the variable and conditional? or even make it a before/after filter?
```suggestion
      update_service_provider
```

Reply:
@zachmargolis , agree, was thinking too much about configurability for it, actually no need for this at this time.